src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.domain
+
+class Lotto private constructor(val lottoNumbers: Set<LottoNumber>) {
+
+    init {
+        val lottoNumbersSize = lottoNumbers.size
+        require(lottoNumbersSize == LOTTO_SIZE)
+    }
+
+    constructor() : this(LottoPreset.shuffled().take(6).sortedBy { it.number }.toSet())
+
+    constructor(vararg number: Int) : this(number.map { LottoNumber(it) }.toSet())
+
+    fun countMatch(winningLotto: Lotto): Int {
+        return lottoNumbers.intersect(winningLotto.lottoNumbers).count()
```

Comment:
saso2

https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
코틀린 공식 문서에 따르면 함수 체인의 경우 아래와 같이 체인별 개행을 해주어야 합니다.
(일반적으로 체인의 첫 번째 호출에는 개행이 있어야 하지만 개행하지 않는 편이 더 합리적이라면 생략해도 괜찮습니다)

```suggestion
        return lottoNumbers
            .intersect(winningLotto.lottoNumbers)
            .count()
```

Reply:
427d34e60d47667a2488c8a99ea910b396444f8a